core/subprojects/packagefiles/icu/meson.build

```diff
@@ -3,7 +3,7 @@ project(
   'c',
   'cpp',
   version: '72.1',
```

Comment:
I thought you'd use ICU 73 (as mentioned in `icu-minimal.wrap` !?

Reply:
fixed